ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.less

```diff
@@ -0,0 +1,69 @@
+@{customElementNamespace}result-form {
```

Comment:
I know this is namespaces to result-form, but since this CSS replicates that of search-form/sharing, might be able to find a way to abstract it all to something that can be shared.

Reply:
This is OBE now because the less files are managing different views now.  I do think a further discussion about reuse is needed